web/js/components/card_wf.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
You should not commit this component, since (a) the naming scheme is wrong and (b) it doesn't use material-ui@next